packages/react-ui/src/Fields/Input.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { box, colors, fonts, spacings } from "../theme";
+
+export const Input = ({ name, icon: Icon, ...props }) => {
+  return (
+    <StyledWrapper>
+      <StyledInput name={name} hasIcon={!!Icon} {...props} />
+      {Icon && (
+        <StyledIcon>
+          <Icon />
+        </StyledIcon>
+      )}
+    </StyledWrapper>
+  );
+};
+
+Input.propTypes = {
+  name: PropTypes.string.required,
+  icon: PropTypes.node
+};
+
+Input.defaultProps = {
+  icon: null
+};
+
+const INPUT_HEIGHT = "5.4rem";
+
+const StyledWrapper = styled.div`
+  position: relative;
+  width: 40rem;
```

Comment:
why 40rem ? Since you pass props to input, I'm not sure we can override the width of the input wrapper

Reply:
Done

Reply:
Je vois pas la modif !